### PR TITLE
Revert "xds: import proto files for xDS LB bootstrap from upstream envoy repository (#6125)"

### PR DESCRIPTION
**Pull request by voidzcy**
This reverts commit d771f254aaeb2e386917c519ed069b62e97310b1.

New xDS bootstrap implementation(#6201) no longer needs the `ApiConfig` proto. So undo the import.

### CODE REVIEW

File: xds/third_party/envoy/import.sh [hunk 1]
```diff
@@ -27,8 +27,6 @@ FILES=(
 envoy/api/v2/auth/cert.proto
 envoy/api/v2/core/address.proto
 envoy/api/v2/core/base.proto
-envoy/api/v2/core/config_source.proto
```
Comment (dapengzhang0) [inline xds/third_party/envoy/import.sh]: `cert.proto` depends on it, so it can not be removed now.

**Comment by voidzcy** [comment]
Closing as `envoy/api/v2/auth/cert.proto` requires these dependencies now, cannot be reverted.

**Comment by dapengzhang0** [comment]
Maybe `grpc_service.proto` can be removed separately.

**Comment by voidzcy** [comment]
> Maybe `grpc_service.proto` can be removed separately.

No, `ApiConfigSource` depends on `grpc_service.proto`. See https://github.com/envoyproxy/envoy/blob/abb14cdd7aa50d0a54a9288420a098314581782a/api/envoy/api/v2/core/config_source.proto#L9

**Pull request closed without merging**